apps/web/src/components/docs/Docs.tsx

```diff
@@ -105,15 +151,38 @@ export const Docs = ({ code = '', description = '', title = '', isLoading, child
           textAlign: 'justify left',
         })}
       >
-        <div>
-          <Flex justify="space-between" align="center">
-            <TitleH1>{title}</TitleH1>
-            {actions}
-          </Flex>
-          <Text>{description}</Text>
-        </div>
+        <When truthy={!isChildDocs}>
+          <div>
+            <Flex justify="space-between" align="center">
+              <TitleH1
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
🤓 **nitpick (non-blocking):** Can just use Title component with variants now‏